index.mako

```diff
@@ -1728,12 +1728,13 @@ like [azure pipelines](#azure-pipelines-example), github actions also uses
 immutable caches:
 
 ```yaml
-    - name: set PY
-      run: echo "::set-env name=PY::$(python -VV | sha256sum | cut -d' ' -f1)"
```

Comment:
a less intrusive change is `>> $GITHUB_ENV`

Reply:
Yeah, updated in a25c534